mopidy/core/actor.py

```diff
@@ -136,6 +140,103 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def setup(self):
```

Comment:
Yes, sorry, I have gone back and fished those out. The reason seems to be to avoid changing the external API in this PR and to concentrate on getting it in. Can just raise another small PR to enable RPC access.